vault/resource_kv_secret_test.go

```diff
@@ -21,13 +23,25 @@ func TestAccKVSecret(t *testing.T) {
 		PreCheck:  func() { testutil.TestAccPreCheck(t) },
 		Steps: []resource.TestStep{
 			{
-				Config: testKVSecretConfig(mount, name),
+				Config: testKVSecretConfig_basic(mount, name),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/%s", mount, name)),
 					resource.TestCheckResourceAttr(resourceName, "data.zip", "zap"),
 					resource.TestCheckResourceAttr(resourceName, "data.foo", "bar"),
 				),
 			},
+			{
+				Config: testKVSecretConfig_basic(mount, name),
```

Comment:
Should not be needed, you can include the extra API check function in the step above.

Reply:
done with [move extra api check to basic test](https://github.com/hashicorp/terraform-provider-vault/pull/1570/commits/13b2fee3fa69c11b6e553bdaf2b33070710c9a9c)